sql-statements/sql-statement-kill.md

```diff
@@ -34,6 +32,7 @@ Query OK, 0 rows affected (0.00 sec)
 
 ## MySQL compatibility
 
+* By design, this statement is not compatible with MySQL by default. This helps prevent against a case of a connection being terminated on the wrong TiDB server, because it is common to place multiple TiDB servers behind a load balancer.
```

Comment:
How to understand `place multiple TiDB servers behind a load balancer'? Could you help explain and make it easier to understand if possible? So it can be translated into Chinese accurately. Thanks!

Reply:
Application -> Load Balancer -> Multiple TiDB Servers.

There is an attempt to fix the problem in https://github.com/pingcap/tidb/pull/17649 - the issue is described in https://github.com/pingcap/tidb/issues/8854